index.d.ts

```diff
@@ -68,6 +70,7 @@ interface Http2Options {
 interface HttpOptions {
   agentOptions?: AgentOptions | SecureAgentOptions;
   requestOptions?: RequestOptions | SecureRequestOptions;
+  agents?: { 'http:': Agent, 'https:': SecureAgent }
```

Comment:
this needs a matching change in the type's tests.

Reply:
added agents to ts test